ans-abs/background.tex

```diff
@@ -1,3 +1,3 @@
 \section{Background}
 
-UIUC uses steam to heat a majority of it's buildings and chilled water to cool them. The campus power plant, Abbott Power Plant (APP), produces 100\% of the campus' steam needs and typically all of the electricity needs. Notable exceptions are peak demand days where the power output of APP, the solar farm, and the wind power purchase agreement (PPA), must be supplemented with electricity purchased from the Mid-continent Independent System Operator (MISO). APP is a co-generation power plant, which means it produces steam and electricity concurrently. In this work we consider only the electricity production. The university has a 10-year PPA with Rail Splitter Wind Farm located in Lincoln, Illinois and a 10-year PPA with Phoenix Solar South Farms, LLC that operates a solar farm on campus \cite{breitweiser_wind_2016,white_solar_2017}. 
\ No newline at end of file
+UIUC uses steam to heat a majority of it's buildings and chilled water to cool them. The campus power plant, Abbott Power Plant (APP), produces 100\% of the campus' steam needs and typically all of the electricity needs. Notable exceptions are peak demand days where the power output of APP, the solar farm, and the wind power purchase agreement (PPA), must be supplemented with electricity purchased from the Mid-continent Independent System Operator (MISO). APP is a co-generation power plant, which means it produces steam and electricity concurrently. In this work we consider the possibility of a retrofitted APP that uses a micro-reactor for cogeneration. Replacing the steam demand with a low-carbon heat source is important for achieving the campus carbon goals outlined in iCAP \cite{isee_illinois_2015}. The university has a 10-year PPA with Rail Splitter Wind Farm located in Lincoln, Illinois and a 10-year PPA with Phoenix Solar South Farms, LLC that operates a solar farm on campus \cite{breitweiser_wind_2016,white_solar_2017}. 
```

Comment:
Two comments:
- Don't write multiple long sentences on the same line.
- In the sixth sentence, you use a vague word: "important" (see writing checklist). Look for a more meaningful replacement.